dags/hivemind_etl_helpers/src/utils/get_communities_data.py

```diff
@@ -134,3 +134,53 @@ def get_discourse_communities() -> list[dict[str, str | datetime]]:
             )
 
     return communities_data
+
+
+def get_google_drive_communities() -> list[dict[str, str | datetime]]:
```

Comment:
The return type could be `list[dict[str, str | datetime | dict]]` because of the client config.